platform_versioned_docs/version-24.1/launch/advanced.mdx

```diff
@@ -27,6 +22,64 @@ For example, modify the **manifest** section to give the pipeline a name and des
     }
 ```
 
+#### Nextflow configuration order of priority
+
+When launcing pipelines in Platform, Nextflow configuration values can be supplied from the `nextflow.config` file in the pipeline repository and the **Nextflow config file** field in the pipeline launch form. If different values of the same configuration parameter are defined, Nextflow parameters defined in the launch form **Nextflow config file** field override the same parameters in your `nextflow.config` file. 
+
+Configuration values set in the **Global Nextflow config** field during compute environment creation are pre-filled in the **Nextflow config file** field during pipeline launch. These pre-filled values from the compute environment can be overridden manually during launch. 
+
+| Nextflow configuration order of priority (highest to lowest) |
+| ------------------------------------------------------------ |
+| Pipeline launch form **Nextflow config file** field |
+| Compute environment **Global Nextflow config** field |
+| Pipeline repository `nextflow.config` file |
+
+For example, if:
+
+1. The `nextflow.config` file in your pipeline repository contains this manifest: 
+
+    ```ini title="Pipeline repository config file"
+    manifest {
+        name = 'A'
+        description = 'Pipeline description A'
+        mainScript = 'main.nf'
+        defaultBranch = 'master'
```

Comment:
I think you can remove these two throughout this PR. We include them in nf-core for paranoia and safety, but they're generally not needed. We may even deprecate `defaultBranch` soon-ish I think.